main/main.cpp

```diff
@@ -1267,8 +1267,16 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 	OS::get_singleton()->_vsync_via_compositor = window_vsync_via_compositor;
 
+	{
+		GLOBAL_DEF_RST_NOVAL("input_devices/tablet/driver", "");
```

Comment:
Also material for a follow up PR, but I'm not sure "tablet driver" is a very good name for this stuff. Isn't this first and foremost about pen/stylus support? This might also relevant on phones or laptops with touch-enabled screens.

*Edit:* Just as I write it I remember that it probably referred to pen/drawing tablets and not Android/iOS/Surface tablets/phablets... so maybe it's fine. Maybe `input_devices/pen_tablet/`?

Reply:
It is for pen/tablet and similar input devices (e.g. setting pressure from some touchpads) input devices. `input_devices/pen_tablet/` seems fine.